test/FunctionalTests/FunctionalTestBase.cs

```diff
@@ -26,9 +26,9 @@ namespace Grpc.AspNetCore.FunctionalTests
 {
     public class FunctionalTestBase
     {
-        private VerifyNoErrorsScope _scope;
+        private VerifyNoErrorsScope? _scope;
 
-        protected GrpcTestFixture<FunctionalTestsWebsite.Startup> Fixture { get; private set; }
```

Comment:
Non-nullability doesn't play well with [OneTimeSetUp] does it.

Reply:
No